sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -352,6 +352,11 @@ public synchronized Client setOperatorWith(AccountId accountId, PublicKey public
         return this;
     }
 
+    public synchronized Client setMaxNodesPerTransaction(int maxNodesPerTransaction) {
+        this.network.setMaxNodesPerTransaction(maxNodesPerTransaction);
+        return this;
+    }
```

Comment:
Should the `maxNodesPerTransaction` on the client be persistent?  EG, if you `setNetwork()` to a new network after setting `maxNodesPerTransaction`, should the `maxNodesPerTransaction` you set on the client carry over to that new network?  That's the behavior I'd probably intuitively expect.  It LOOKS like this sets a property of the client, but it actually doesn't at all, it sets a property of the network that it _happens to be using_ at that time.

Is this just a convenience function to use while building a client so users don't need to `client.network.setMaxNodesPerTransaction()` on a separate line?  I'm not sure if the potential for confusion is worth that.  I would remove it entirely, I think.